pkg/imagestore/imagestore.go

```diff
@@ -19,28 +19,28 @@ var DefaultVersions = []map[string]string{
 	{
 		"openshift_version": "4.6",
 		"cpu_architecture":  "x86_64",
-		"url":               "https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.6/4.6.8/rhcos-4.6.8-x86_64-live.x86_64.iso",
-		"rootfs_url":        "https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.6/4.6.8/rhcos-live-rootfs.x86_64.img",
+		"url":               "https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.6/4.6.8/rhcos-4.6.8-x86_64-live.x86_64.iso",
```

Comment:
nice:) would be clean when using '/arm64/dependencies/' path.

Reply:
:+1: that's the idea.